sky/skylet/job_lib.py

```diff
@@ -436,7 +435,7 @@ def _get_jobs(username: Optional[str],
             WHERE status IN ({','.join(['?'] * len(status_list))})
             AND submitted_at <= (?)
```

Comment:
Since we are not using the `gap` now, let's remove this line `AND submitted_at <= (?)` and the later `time.time()` for simplicity.